css/style.css

```diff
@@ -4,8 +4,251 @@
 body {
     font-family: 'Roboto', sans-serif;
     -webkit-font-smoothing: antialiased;
+    overflow-x: hidden;
 }
 
+header.logo{
+    width: 30px;
+    height: 30px;
+    position: relative;
+    top: 30px;left: 120px;
+}
+
+.navbar{
+  color:rgb(88, 69, 69) 
+  
+   
+}
+.hamburger{
+  display: none;
+}
+
+  ul li a {
+    list-style-type: none;
+    font-size: 15px;
+    position: relative;
+    top: 0px;right: 100px;
+    text-decoration: none;
+    
+}
+  ul li a:hover
+  {
+  color: rgb(235, 106, 31);
+  }
+
+  li {
+    display: inline;
+    padding: 7px;
+    margin: 10px;
+    margin-top: 0;
+    float: right;
+    
+  }
+
+  .photo{
+    background-image: url(/img/first-background.jpg);
+    background-size:cover;
+    background-position: center;
+    background-repeat: no-repeat;
+    margin: 0;
+    padding: 0;
+    width: 1500px;
+    height: 1000px;
+    align-items: center; 
+    position: relative;
+    top: 0px;left: 30px;
+    
+    
+  }
+
+.photo h3,
+h4 {
+  color: white;
+  text-align: center;
+  position: relative;
+  top: 270px;left: 200px;
+  vertical-align: middle;
+ 
+}
+
+.photo h3 {
+  font-size: 70px;
+  font-weight: 100;
+  margin-bottom: 10px;
+  position: relative;
+  top: 360px;left: 15px;
+}
+
+.photo h4 {
+  font-size: 50px;
+  font-weight: 50;
+  margin-top: 50px:
+  position: relative;
+  top: 310px;left: -30px;
+}
+
+.learn{
+  
+  line-height: 2.8;
+  padding: 0 20px;
+  font-size: 1rem;
+  text-align: center;
+  color: #fff;
+  border-radius: 5px;
+  background-color: rgb(209, 115, 52);
+  position: relative;
+  top: -430px;left: 700px;
+}
+
+.icon p{
+  text-align: center;
+  font-size: 30px;
+}
+
+.icon img{
+  width: 180px;
+  height: 80px;
+  position: relative;
+  left: 500px;
+ 
+}
+.specs p{
+  display: inline;
+  position: relative;
+  left: 480px; 
+  padding: 20px;
+  margin-top: 50px;
+  font-size: 15px;
+}
+
+ footer h5 {
+    text-align: center;
+    margin-top: 60px;
+  
+  }
+footer img{
+  
+    width: 15px;
+    height: 15px;
+    margin: 10px;
+    margin-top: 5px;
+    position: relative;
+    left:725px;
+  }
+  footer p {
+    margin-top: 20px;
+    text-align: center;
+    font-size: 12px;
+  }
+
+  /* MOBILE*/
+
+@media (max-width: 600px) {
+
+  header.logo{
+    width: 30px;
+    height: 30px;
+    position: relative;
+    top: 10px;left: 15px;
+}
+  .navbar {
+    display: none;
+  }
+.hamburger{
+  display: block;
+  float: right;
+  padding: 0px;
+
+}
+.photo{
+  background-image: url(/img/first-background.jpg);
+  background-size:cover;
+  background-position: center;
+  background-repeat: no-repeat;
+  width: 100%;
+  height: 600px;
+  position: relative;
```

Comment:
You don't need to repeat rules that are the same as for desktop. Only put what is _different_ in your media query (for example the height)